arch/arm/src/arm/Toolchain.defs

```diff
@@ -83,6 +83,26 @@ ifeq ($(CONFIG_MM_KASAN),y)
   ARCHOPTIMIZATION += -fsanitize=kernel-address
 endif
 
```

Comment:
let's add the master switch:
ifeq ($(CONFIG_MM_UBSAN),y)
  ARCHOPTIMIZATION += -fsanitize=undefined
endif